src/index.html

```diff
@@ -17,11 +17,333 @@
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__link">buy</button>
+      </div>
+
+      <div class="card card--6">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-number">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__link">buy</button>
+      </div>
+
+      <div class="card card--7">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-number">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__link">buy</button>
+      </div>
+
+      <div class="card card--8">
+        <img
+          src="./images/imac.jpeg"
+          alt="imac"
+          class="card__image"
+        />
+        <h2 class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="card__code">Product code: 195434</p>
+        <div class="card__reviews">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews-number">Reviews: 5</p>
+        </div>
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-cost">$2,199</p>
+        </div>
+
+        <button class="card__link">buy</button>
+      </div>
```

Comment:
The cards in your catalog are very similar. This is a great opportunity to use a template engine to avoid repeating the same code for each card. This would make your code shorter, more readable and easier to maintain.